src/locales/zh-hans.json

```diff
@@ -0,0 +1,67 @@
+{
+  "skip_to_main": "Skip to main content",
+  "index": {
+    "title": "Build the next Ruby for Wasm with Artichoke",
+    "description": [
+      "Bundle Ruby applications into a single Wasm executable with Artichoke, a Ruby made with Rust. The modular interpreter enables running untrusted code.",
+      "Artichoke is pre-release software. We're looking for contributors to build a Ruby for pure Wasm targets."
+    ],
+    "contribute": "贡献",
+    "current": "Currently",
+    "previous_releases": "Previous releases",
+    "install": {
+      "title": "安装",
+      "content": "Install Artichoke Ruby nightly builds with rbenv, Docker, or Cargo.",
+      "hint": "Nightly installs include artichoke (ruby) and airb (irb) binaries. The Artichoke Ruby Core and Standard Library runtime is bundled in these two artifacts.",
+      "guide": "阅读安装说明"
+    },
+    "subscribe": {
+      "title": "订阅",
+      "content": "Stay up to date with project updates on Twitter",
+      "hint": "We'll tweet about Rust, crates we're developing, and new releases."
+    }
+  },
+  "install": {
+    "title": "Install Artichoke – Artichoke Ruby",
+    "install": "安装Artichoke Ruby",
+    "targets": "Artichoke Ruby installs on Linux, macOS, and Windows. New builds are pushed nightly.",
+    "try": "Try out a build using one of these installation methods.",
+    "description": "Install Artichoke Ruby, a Ruby made with Rust.",
+    "nightly": {
+      "go": "每夜构建",
+      "title": "Pre-compiled nightly builds",
+      "content": "Download a prebuilt binary from artichoke/nightly. Binaries are available for Linux, Linux/musl, macOS, and Windows.",
+      "hint": "These daily binaries track the latest trunk commit of Artichoke. Binaries are also distributed through",
+      "button": "Previous Releases"
+    },
+    "docker": {
+      "go": "Docker",
+      "title": "Docker镜像",
+      "content": "Nightly builds of Artichoke are pushed to Docker Hub. Images are built on Ubuntu 18.04, Debian Buster Slim, and Alpine 3.",
+      "hint": "These daily images track the latest trunk commit of Artichoke.",
+      "button": "On Docker Hub"
+    },
+    "cargo": {
+      "go": "Cargo",
+      "title": "使用Cargo",
+      "content": "Artichoke builds and installs using Cargo, the Rust package manager.",
+      "hint": "Building with Cargo compiles the latest trunk commit of Artichoke.",
+      "button": "构建指南"
+    }
+  },
+  "footer": {
+    "project": "项目",
+    "organization": "组织",
+    "vision": "Vision",
+    "resources": "资源",
+    "file_bug": "提交Bug",
+    "docs": "Crate文档",
+    "social": "Social"
+  },
+  "nav": {
+    "home": "主页",
+    "install": "安装",
+    "code": "代码",
+    "docs": "文档"
+  }
+}
```

Comment:
I think prettier won't like the missing newline here.

What does `npm run fmt` do?